src/main/java/com/treasuredata/android/TreasureData.java

```diff
@@ -846,6 +863,58 @@ public void fetchUserSegments(List<String> profilesTokens, Map<String, String> k
         }
     }
 
+    private String defaultValueTableKey(String database, String table) {
+        String _database = database == null ? "" : database;
+        String _table = table == null ? "" : table;
+        return String.format("%s.%s", _database, _table);
+    }
+
+    /**
+     * Set default `value` for `key` in all new events targeting `database` and `table`.
+     * When `database` and/or `table` parameters are null, the null parameter acts like a wild card that allows to set specified key value pair to new events added to any database (if `database` is null) and/or to any table (if `table` is null).
+     * For example, if you pass null to both `database` and `table` parameters, all new events will have specified default value.
+     * @param database the database to set default value to. If null, specified table of any database will have new events with the added default value.
+     * @param table the table to set default value to. If null, any table of specified database will have new events with the added default value.
+     * @param key the event's key that default value is set to, corresponding to column in table.
+     * @param value default value for `key`
+     */
+    public void setDefaultValue(String database, String table, String key, Object value) {
```

Comment:
Should we consider provide overloads instead of nullable parameters?
Now that thinking about this, the positioning database & table first might give the wrong impression that `key` and `value` are the optional ones. But I think it's an unavoidable trade-off we have to make (to be consistent with `addEvent`)

Reply:
I prefer this explicit null rather than implicit. I don't think it's much of consistency issue with addEvent:
- Since for addEvent, user will have to set default table and default database beforehand.
- addEvent use frequently, hence it benefits from shorter but implicit method signature.
- setDefaultValue mostly used only once when configuring SDK, making multiple setDefaultValue method calls look prettier together.

Reply:
No objection.